### PR TITLE
[WNMGDS-1855] Add box-sizing rule. Remove focus sizing styles - unneeded.

### PR DESCRIPTION
**Pull request by zarahzachz**
## Summary

This work folds in a local style from the Windowshop/Enroll app where small checkbox icons weren't displaying correctly. 

It turned out that app's CSS reseat included a `box-sizing: initial` for all elements, (`*, *::before, *::after { box-sizing: initial; }`) that was breaking the layout of our CSS. They fixed it by adding a local `box-sizing: content-box` to the checkbox styles. I migrated this style into our design system.

While testing this change, I discovered another bug:
Screen Shot 2022-10-17 at 11 05 43 AM

Small checkboxes increase in size on focus. After playing around with the style causing this, I was able to determine we could safely remove the offending CSS. Tested in Chrome, Firefox, and Safari and removing this style had no impact on any checkboxes in our design system.

## How to test

Run Storybook locally and navigate to: http://localhost:6006/?path=/story/components-choicelist--small-option&args=type:checkbox

Clicking new checkboxes (moving focus around) should persist the small sizing.

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title`

### If this is a change to code:

- [ ] Created or updated unit tests to cover the logic added
- [ ] If necessary, updated unit-test snapshots (`yarn test:unit:update`) and browser-test snapshots (`yarn test:browser:update`)

### If this is a change to documentation:

- [ ] Checked for spelling and grammatical errors

### CODE REVIEW

File: packages/design-system/src/styles/components/_Choice.scss [hunk 1]
```diff
@@ -140,12 +145,6 @@ label,
     &::after {
       opacity: 1;
     }
-
```
Comment (forrestbaer) [inline packages/design-system/src/styles/components/_Choice.scss:143]: Yay removed a calc! 🎉

**Review by forrestbaer — COMMENTED** [review]
Looks good, were there any changes in visual regression?

**Comment by zarahzachz** [comment]
> Looks good, were there any changes in visual regression?

I ran the command locally and didn't find any. Obviously, there were visual changes to removing the focus calc sizing, but we don't have vis regression tests for interactive states like focus so it didn't flag it.

**Review by forrestbaer — APPROVED** [review]

File: packages/design-system/src/styles/components/_Choice.scss (hunk 1 above)
Comment (zarahzachz) [reply packages/design-system/src/styles/components/_Choice.scss:143]: Oh, looks like Mgov has overrides for this style. Pushing up a fix addressing that too.

Comment (pwolfert) [reply packages/design-system/src/styles/components/_Choice.scss:143]: Hey, it looks nice!

**Pull request merged**